src/styles/blocks/button.scss

```
@@ -0,0 +1,21 @@
+.button {
```

and move to card.scss
```suggestion
.card__button {
```